.gitlab/pipelines/stages/tests/smoke_tests.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+# shellcheck disable=SC1090
+source "$(bazel info workspace)/.gitlab/pipelines/config/config.sh"
+
+# Trigger smoke-tests.
+bazel run //testing/smoke_tests
```

Comment:
It might be nice if `bazel run //testing/smoke_tests` would also display the logs…